custom_components/tuya_local/devices/srne_energy_storage_battery.yaml

```diff
@@ -0,0 +1,310 @@
+name: LiFP battery
+products:
+  - id: l2ak4zqufuarm0j3
+    manufacturer : SRNE
+    model: SR-SE05B
+entities:
+  - entity: sensor
+    class: battery
+    translation_key: Charge
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    class: voltage
+    translation_key: Battery voltage
```

Comment:
No translation key by this name exists, make it a name, or even just leave it out and use the default "voltage" class translation, since it is the main voltage sensor this device has so it is not ambiguous (it will be obvious from the value that it is a battery voltage, not a mains voltage).